README.md

```diff
@@ -130,6 +131,7 @@ Jump to [0-9](#0-9) | [A](#a) | [B](#b) | [C](#c) | [D](#d) | [E](#e) | [F](#f)
 - [YOURLS GWall Change Title](https://github.com/gioxx/YOURLS-GWallChangeTitle) - Allows you to change the title of the YOURLS website (the html `<title>` tag).
 - [Change Password](https://github.com/ozh/yourls-change-password) ☑️ - Change your password from within YOURLS instead of editing the config file.
 - [Change Password](https://github.com/vvanasten/YOURLS-Change-Password) - Allow users to change their password via the administration interface.
+- [Change Shortlink User](https://git.oldgate.org/Sophia/yourls-change-shortlink-user) - Allows admin to change the user of a shortlink, including setting it to nothing.
```

Comment:
I'm a bit confused by this one, as YOURLS doesn't store the username that created a given shortlink.

Even if it did, you should use the URL table constant `YOURLS_DB_TABLE_URL` instead of hard-coding the table name `yourls_url` in the SQL.

As written, this plugin looks more likely to cause confusion than to be useful to a general user. (And it claims to be tested with YOURLS "v1.19.2"?)

Reply:
I'm 99% sure that YOURLS does save the user for links created by logged-in users, as I was able to see this on my dev server.

For your second point, I'll need to look into it further.

And I disagree with your third statement. This feature is aimed at admins who have multiple users and might rely on a system of link control. It can be used to correct link mistakes made by users, which is valuable in that context.

Reply:
> I'm 99% sure that YOURLS does save the user for links created by logged-in users, as I was able to see this on my dev server.

There is no user field in the URL table:

https://github.com/YOURLS/YOURLS/blob/0d59b83a8c0ee0061143b0c660096c1d816036eb/includes/functions-install.php#L217-L228

A plugin called Separate Users adds one, though:

https://github.com/ianbarber/Yourls-Separate-Users/blob/c05feae0784d43366d49347d45979776c07bfa75/plugins/separate-users/plugin.php#L49-L68

> For your second point, I'll need to look into it further.

That plugin also uses the constant I mentioned, `YOURLS_DB_TABLE_URL` :)

> And I disagree with your third statement. This feature is aimed at admins who have multiple users and might rely on a system of link control. It can be used to correct link mistakes made by users, which is valuable in that context.

I agree that it sounds useful! But there is no user tracking in YOURLS core, and that's why I think it would confuse users without additional context. If you want to list a plugin that extends another plugin, that's great—it should just be clear that your code depends on another third-party component.

It really sounds like your test server either has or once had the Separate Users plugin installed, because it does modify the core URL table as linked above. It also leaves that added column behind when deactivated. _(IMO it _should_ use a separate table with `url.keyword` as a foreign key, but that's not your problem 😺)_

> Also, the version number is 1.19.2 was a mistake, I play Minecraft and put in a Minecraft version number lol

Thought it was a typo, but figured it would be worth correcting so potential users won't think you somehow wrote a plugin in the future 😂

Reply:
OMFG, so I can explain this: [Auth Manager Plus](https://github.com/joshp23/YOURLS-AuthMgrPlus) is built into a plugin I made for my service. That's 100% my fault. I've just been using that plugin since I started using YOURLS over three years ago. 

Is it okay to remove this one, as I made it for myself, and no harm is done,

Sorry for the mix-up, this is what happens when you make things purpose-built for yourself lol.

Reply:
> Is it okay to remove this one, as I made it for myself, and no harm is done,

Totally fine to remove. I can modify the PR for you if you want, or you can remove this line yourself.

Don't worry too much about the commit history; we squash-and-merge this repo. :)

Reply:
That works for me, if you could remove it please :)